go.mod

```diff
@@ -37,3 +37,5 @@ replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-ho
 
 // FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/182
 replace github.com/peggyjv/gravity-bridge/module => github.com/yihuang/gravity-bridge/module v0.1.22-0.20210908191543-71021d1bfff2
+
+replace github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.4.2-0.20210914024651-30df99098708
```

Comment:
could you use `crypto-org-chain/ethermint` instead?

it would be easier for maintenance

Reply:
Yeah, sure. currently I open this draft PR to run all the CI checks, I plan to update this after the upstream PR get merged.

Reply:
similarly for the gravity bridge -- if we anticipate the custom patches will keep hanging for a while, it'll be better to consolidate them under `crypto-org-chain/gravity-bridge/module` instead of keeping it spread on different personal forks

Reply:
> similarly for the gravity bridge -- if we anticipate the custom patches will keep hanging for a while, it'll be better to consolidate them under `crypto-org-chain/gravity-bridge/module` instead of keeping it spread on different personal forks

makes sense, opened issue for that: https://github.com/crypto-org-chain/cronos/issues/71.

Reply:
removed the replace, upstream PR merged.